k2/csrc/fsa_utils.cu

```diff
@@ -477,6 +506,8 @@ static Fsa OpenFstTransducerFromStream(std::istringstream &is,
   }
 
   K2_CHECK(is.eof());
+  K2_CHECK_GT(max_state, 0) << "Invalid input stream: "
+                            << "\nOpenFST transducer expects valid states.";
 
```

Comment:
.. BTW in order to support multiple aux_labels, I would be OK to just make the `aux_labels` arg to this function an array2, which would probably make the code changes quite minimal.
We can assume that the caller knows how many aux_labels there are.